src/NLU.DevOps.CommandLine/Compare/CompareCommand.cs

```diff
@@ -38,8 +38,10 @@ public static int Run(CompareOptions options)
                 var compareResults = TestCaseSource.GetNLUCompareResults(expectedUtterances, actualUtterances);
                 var metadataPath = options.OutputFolder != null ? Path.Combine(options.OutputFolder, TestMetadataFileName) : TestMetadataFileName;
                 var statisticsPath = options.OutputFolder != null ? Path.Combine(options.OutputFolder, TestStatisticsFileName) : TestStatisticsFileName;
+
                 Write(metadataPath, compareResults.TestCases);
                 File.WriteAllText(statisticsPath, JObject.FromObject(compareResults.Statistics).ToString());
+                compareResults.PrintResults();
```

Comment:
Didn't we just merge this? Can you rebase this PR?

Reply:
fixed it.